apps/wakunode2/app.nim

```diff
@@ -590,6 +591,8 @@ proc startRestServer(app: App, address: ValidIpAddress, port: Port, conf: WakuNo
   ## Store REST API
   installStoreApiHandlers(server.router, app.node)
 
+  installLightPushPostPushRequestHandler(server.router, app.node)
```

Comment:
That name! :rofl:  This is getting ridiculous. "installLightPushHandler" would be fine.

Reply:
Oh well, after re-reading it... yes, absolutely right. Will fix it.